Standards/scs-0218-v1-container-registry-for-scs-standard-implementation.md

```diff
@@ -377,13 +245,51 @@ storage drivers instead so that each customer gets their own separate storage ba
+| Registry replication                | ✓                                       | ✓                                                                   | ✓                             |
+| Proxy cache                         | ✓                                       | ✓ Feature is in the technology preview stage (non production ready) | ✗                             |
+| Quota management                    | ✓ Based on storage consumption          | ✓ Based on storage consumption                                      | ✗                             |
+| Garbage collection                  | ✓ Non-blocking                          | ✓ Non-blocking                                                      | ✗                             |
+| Retention policy                    | ✓ Multiple tag retention rules          | ✓ Only tag expiration rules                                         | ✗                             |
+| Additional supported artifacts      | ✗ (only OCI artifacts)                  | ✗ (only OCI artifacts)                                              | ✓ Maven, YUM                  |
+| Integration possibilities           | ✓ Dragonfly (P2P), Kraken (P2P)         | ✗                                                                   | ✓ Harbor, Nydus, eStargz      |
+| Deployment capabilities             | ✓ Docker-compose, Helm chart, Operator  | ✓ Docker-compose, Operator                                          | ✓ Docker-compose, Helm chart  |
+| Administration capabilities         | ✓ Terraform, CRDs, Client libraries     | ✓ Ansible, Client libraries                                         | ✓ Client libraries            |
+
+Notes:
+
+- Automation: Harbor should support webhooks following CloudEvents spec in the [next release](https://github.com/goharbor/harbor/issues/17748)
+- Content Trust and Validation: Harbor announced the deprecation of [Notary](https://github.com/goharbor/harbor/discussions/16612)
+  integration, hence it is not mentioned in the table
+- Multi-tenancy: Harbor, Quay, as well as Dragonfly, operates on a single storage
+  backend (e.g. S3), i.e. the storage of container images is shared between tenants
+- Additional supported artifacts: Harbor announced the deprecation of [Chartmuseum](https://github.com/goharbor/harbor/discussions/15057)
+  integration, hence it is not mentioned in the table
+
 ## Decision
 
-Based on the research and conclusion above we've decided to use the **Harbor** project
-as a container registry implementation for the SCS ecosystem.
+Based on the requirements laid out in ["Requirements for container registries"], the OSS health check
+and the possible software solutions presented in this document, it was decided to use the **Harbor** project
+as the container registry for the SCS reference implementation and ecosystem.
```

Comment:
```suggestion
as the container registry for the SCS reference implementation.
```

Reply:
Changed in https://github.com/SovereignCloudStack/standards/pull/458/commits/e73ed463c4d03e128dcfa7f1437bfa7131de3f93